admin_guide/master_node_configuration.adoc

```diff
@@ -268,7 +268,9 @@ kind: NodeConfig
 masterKubeConfig: node.kubeconfig
 networkPluginName: ""
 nodeName: node1.example.com
-podManifestConfig: null
+podManifestConfig: <1>
+  path: "/tmp/path/to/pod-manifest-file" <2>
```

Comment:
I would remove /tmp from the path - the implications of that could be misleading.
`/path/to/pod-manifest-file` should be adequate.